okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -1216,6 +1221,7 @@ public void settings(boolean clearPrevious, Settings settings) {
 
     @Override
     public void ping(boolean ack, int payload1, int payload2) {
+      logger.logPing(OkHttpFrameLogger.Direction.INBOUND, ((long)payload1 << 32) | (payload2 & 0xFFFFFFFFL));
```

Comment:
If this bit operation is reused, declare a static method and reuse the method.

Reply:
This is because in HTTP/2 spec, ping payload is a 8 octets and in our method, the payload is passed as two `int`. I am trying to preserve the nature of HTTP/2 spec and therefore concat the two `int`. I don't think it's necessary to have a separate method to do that.

Reply:
There is exactly the same code, move the line below
```
long ackPayload = (((long) payload1) << 32) | (payload2 & 0xffffffffL);
```
to above.